content/ember/v4/link-to-tag-name.md

```diff
@@ -0,0 +1,12 @@
+---
+id: ember.link-to.tag-name
+title: Passing tagName to LinkTo
+until: '5.0.0'
+since: '4.0.0'
```

Comment:
This file should be deleted. We already have an entry: https://deprecations.emberjs.com/v3.x#toc_ember-built-in-components-legacy-arguments

And a PR: https://github.com/emberjs/ember.js/pull/19747